JavaErickHernandez/Day3/Lists.java

```diff
@@ -0,0 +1,116 @@
+package bootcamp;
+import java.util.*;;
+
+public class Lists {
+	static List<Integer> numbers = new ArrayList<>();
+	static String numberOfInputs = "Enter the size of the list: ";
+	static String numberOfInputsFinal = "Your number of Inputs: ";
+	static String tryAgainMessage = "Try Again";
+	static String enterInput = "Enter your input: ";
+	static String enterX = "Enter index: ";
+	static String enterQueries = "Enter the number of Queries: ";
+	static String enterOption = "Type Insert or Delete to continue: ";
+	static String warning = "Input must not be greater than List size: ";
+	static String error = "Error!";
+
+	public static void main(String[] args) {
+		numberOfRows();
+	}
+
+	public static void numberOfRows() {
+		System.out.println(numberOfInputs);
+		int finalNumberOfRows;
+		do {
+			finalNumberOfRows = isNumber();
+		} while (finalNumberOfRows <= 0 && finalNumberOfRows >= 4000);
+		System.out.println(numberOfInputsFinal + finalNumberOfRows);
+		fillList(finalNumberOfRows);
+	}
+
+	public static void fillList(int inputs) {
+		for (int i = 1; i <= inputs; i++) { // column
```

Comment:
delete the comment, if you need that information I suggest you change the name of the method to "fillColimnList"